vizro-core/hatch.toml

```diff
@@ -14,9 +14,11 @@ matrix.python.features = [
 dependencies = ["scriv"]
 detached = true
 scripts = {add = "scriv create --add", collect = ["scriv collect --add", "- hatch run lint:lint --files=CHANGELOG.md"]}
+installer = "uv"
 
 [envs.default]
 dependencies = [
+  "pip",
```

Comment:
Why do we need this? I tried removing it and creating a fresh environment already had `pip` installed.

Reply:
Hmm no, this worked fine for me straight away. Doing `hatch env prune` and `hatch run pip freeze` works without the explicit `pip` dependency.

Reply:
I guess the source of truth here will be whether it runs on CI without the dependency listed?

Reply:
Ah, I think you are right right here and `pip` doesn't come with the environment indeed. Looks like my computer was just picking up a global `pip`.

That said, why do we actually want `pip`? Is it just for the `pip freeze` steps in CI?

Reply:
That and I do find myself sometimes using it when I am not sure why my `hatch` environment atm is actually at. So I think that is quite a natural command

Reply:
Cool. Over time maybe we should try and remove this to become fully `uv`-native but doesn't matter really.

I guess you might need to add `pip` as a dependency to some of the isolated environments also btw.